var/spack/repos/builtin/packages/py-tornado/package.py

```diff
@@ -10,13 +10,24 @@ class PyTornado(PythonPackage):
     """Tornado is a Python web framework and asynchronous networking
     library."""
     homepage = "https://github.com/tornadoweb/tornado"
-    url      = "https://github.com/tornadoweb/tornado/archive/v4.4.0.tar.gz"
+    url      = "https://github.com/tornadoweb/tornado/archive/v6.0.3.tar.gz"
 
+    version('6.0.3',   sha256='a97ac3b8c95867e534b48cb6fbbf156f5ca5b20c423bb06894c17b240d7a18fc')
+    version('6.0.2',   sha256='a5f744b5516d8a94171db5e02096dbd5169a5df2034a2b562a150b53b6d4db70')
+    version('6.0.1',   sha256='02e41f5d7a55b78e002675ccdbbd181a0ce9b5f857b5a717227521f10a11c583')
+    version('6.0.0b1', sha256='54d20962d0bc8c6a05792c39349de9ad5d8f51605e1c071cc68a0241622bfa18')
```

Comment:
You don't have to add all of these releases if you don't want to. I usually just add the latest version. At the very least, I wouldn't add these beta versions.